src/common/records.lua

```diff
@@ -7,27 +7,41 @@ Records = Records
 		["@"] = {
 			transactionId = "-k7t8xMoB8hW482609Z9F4bTFMC3MnuW8bTvTyT8pFI",
 			ttlSeconds = 900,
+			priority = nil,
 		},
 	}
 
 --- Set a record in the Records of the ANT.
 ---@param name string The name of the record.
 ---@param transactionId string The transaction ID of the record.
 ---@param ttlSeconds number The time-to-live in seconds for the record.
+---@param priority integer|nil The sort order of the record
 ---@return Record
-function records.setRecord(name, transactionId, ttlSeconds)
+function records.setRecord(name, transactionId, ttlSeconds, priority)
 	utils.validateUndername(name)
 	assert(utils.isValidArweaveAddress(transactionId), "Invalid Arweave ID")
 	utils.validateTTLSeconds(ttlSeconds)
+	assert(
+		priority == nil or math.type(priority) == "integer",
```

Comment:
```suggestion
		priority == nil or priority and name != '@' and priority > 0 and math.type(priority)=="integer",
```

i.e. priority must be an integer greater than 0 when name is not '@'

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/77/commits/a32e90a50330ea185fc2469eb1f1f4d574e9b791